wordpress-mobile/WordPress-iOS
# Use PostRepository.getPost in SearchManager

What says in the title. The new `PostRepository` type was introduced in #21419.

## Test Instructions

You may need a real device for this, because simulator's Spotlight search may not work properly.
1. Go to "Posts" list to make sure posts are indexed by Spotlight.
2. Bring up Spotlight search box and search a post by typing its title.
3. There should be a Jetpack/WordPress result corresponding to the search
4. Tap the result in the Spotlight search result.
5. The app should be brought up and redirects to the post reader screen.

## Regression Notes
1. Potential unintended areas of impact
None.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
The "Test Instructions"

3. What automated tests I added (or what prevented me from doing so)
None

PR submission checklist:


UI Changes testing checklist: N/A
